layouts/taxonomy/tag.html

```diff
@@ -7,7 +7,7 @@
     
   <div class="post-list__container">
     <div class="tag__header">
-      <a href="/blog">All posts</a><span class="separator">/</span>
+      <a href="{{ .Site.Params.blogurl }}">All posts</a><span class="separator">/</span>
```

Comment:
Let's have a default option here so that this doesn't break sites that don't have that param specified.
`<a href="{{ .Site.Params.blogurl | default "/blog" }}">All posts</a><span class="separator">/</span>`